src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -170,98 +170,96 @@
     </section>
 
     <!-- Sample Templates -->
-    <template>
```

Comment:
Was getting a console warning: `"COMPILER_NATIVE_TEMPLATE deprecation with no special directives will render as a native template element instead of its inner content in Vue 3."`

I just removed the template tag here.

Reply:
Personally I liked this template tag as it encapsulated a section consistent with other sections in here.

It's hacky but could we do `<template v-if="true">` ? Or is there another to have a code block?

Reply:
^^ This was to faciliate code-folding of the sample templates (multiple sections).

Reply:
Yup, I tried returning the template and for some reason lint isn't complaining about it now. Also, I can't see the console warning now. I'll be returning this back Sev.

Reply:
You reverted this. Resolving.